test/config/AppConfigSpec.scala

```diff
@@ -97,6 +97,16 @@ class AppConfigSpec extends SpecBase with ShouldMatchers {
     }
   }
 
+  "DataStore" should {
```

Comment:
Better to say `"customsDataStore" should {` and then update description on line 101